app/helpers/media_helper.rb

```diff
@@ -109,7 +109,7 @@ def media_sorts_select(purpose)
     return add_prompt(Medium.select_quizzables) if purpose == 'quiz'
     return Medium.select_question if purpose == 'clicker'
     return add_prompt(Medium.select_importables) if purpose == 'import'
-    return add_prompt(Medium.select_generic) if !current_user.admin?
+    return add_prompt(Medium.select_generic) if !current_user.admin_or_editor?
```

Comment:
rubocop suggest this:

```ruby
unless current_user.admin_or_editor?
      return y(Medium.select_generic)
end
```

It suggested
```ruby
return add_prompt(Medium.select_generic) unless current_user.admin_or_editor?
```
first, but the line was too long then, so it favored the former.